tensorflow/java/src/main/java/org/tensorflow/Graph.java

```diff
@@ -77,6 +80,22 @@ public Operation operation(String name) {
   }
 
   /**
+   * Returns all operations that exist in the Graph.
+   */
+  public List<Operation> getOperations() {
```

Comment:
Traditionally (with quotes around that), some Iterator implementations will throw `ConcurrentModificationException` should the underlying collection change between construction time and a given invocation of `next()`. To have this iterator do that would be a bit of a hugger-ma-rug; so i'd vote for noting its absence in javadocs and moving on. Should this functionality become a demand for prime-time, we could go back and add functionality to allow such detection and a subsequent throw.